app/controllers/application_controller/ci_processing.rb

```diff
@@ -26,10 +26,11 @@ def set_ownership
     if !session[:checked_items].nil? && @lastaction == "set_checked_items"
       recs = session[:checked_items]
     else
-      recs = find_checked_items
+      recs = find_checked_items_with_rbac(get_class_from_controller_param(params[:controller]))
```

Comment:
Reamed 'find_checked_items_with_rbac' to 'find_checked_ids_with_rbac' https://github.com/ManageIQ/manageiq-ui-classic/pull/779/commits/c45960eece96603dfe4a534afa62e5f5a7ea0753